nevergrad/functions/gym/gym_multi.py

```diff
@@ -0,0 +1,924 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import copy
+import numpy as np
+import os
+import typing as tp
+import gym
+
+compiler_gym_present = True
+try:
+    import compiler_gym  # pylint: disable=unused-import
+    from compiler_gym.envs import CompilerEnv
+except ImportError:
+    compiler_gym_present = False
```

Comment:
move the import to the function, raise ng.errors.UnsupportedExperiment if not working

Reply:
done.